wremnants/qcdScaleByHelicity_helper.py

```diff
@@ -9,16 +9,16 @@
 
 def makeQCDScaleByHelicityHelper(input_path=f"{data_dir}/angularCoefficients"):
     charge_axis = hist.axis.Regular(
-        2, -2, 2, name="charge", underflow=False, overflow=False
+        2, -2, 2, name="chargeVgen", underflow=False, overflow=False
     )
     mur_axis = hist.axis.Regular(
-        3, 0, 3, name="muRDownNomUp", underflow=False, overflow=False
+        3, -1.5, 1.5, name="muRDownNomUp", underflow=False, overflow=False
     )
     muf_axis = hist.axis.Regular(
-        3, 0, 3, name="muFDownNomUp", underflow=False, overflow=False
+        3, -1.5, 1.5, name="muFDownNomUp", underflow=False, overflow=False
```

Comment:
It might make sense to structure the axis such that ax[0] = ax[0.5j] and ax[2] = ax[2j] in which case (3, 0, 3) makes sense (or 3, 0.5, 2.5 for example)

Reply:
Hmm, yes I see your point. I'ts a bit awkward but this could also be Variable([0.25, 0.75, 1.25, 2.75])